include/ola/base/Macro.h

```diff
@@ -63,6 +63,30 @@
 #endif  // __GNUC__
 
 /**
+ * @def OLA_FALLTROUGH
+ * @brief Mark switch cases as falltrough when required
+ *
+ * @examplepara
+ *   @code
+ *   switch (cond) {
+ *     case 'foo':
+ *       doFoo();
+ *       //after foo falltrough to bar because $REASON
+ *       OLA_FALLTROUGH
+ *     case 'bar':
+ *   @endcode
+ */
+#ifdef __GNUC__
+#if __GNUC__ >= 7
+#define OLA_FALLTROUGH __attribute__ ((fallthrough));
```

Comment:
True. Perhaps just add it as a future todo/comment for why we aren't using the [[]] variant.